apps/mail/components/create/template-button.tsx

```
@@ -0,0 +1,260 @@
+                            duration: 10000,
+                            action: {
+                              label: 'Delete',
+                              onClick: () => handleDeleteTemplate(t.id, t.name),
+                            },
+                            className: 'pointer-events-auto',
+                            style: {
+                              pointerEvents: 'auto',
+                            },
+                          });
+                        }}
+                      >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)